pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -675,18 +696,30 @@ func isPlatformScanPodRunning(r *ReconcileComplianceScan, scanInstance *compv1al
 }
 
 func isPodRunning(r *ReconcileComplianceScan, podName, namespace string, logger logr.Logger) (bool, error) {
+	podlogger := logger.WithValues("Pod.Name", podName)
```

Comment:
Out of curiosity, what does this do? Does it add a key-value pair to the logger that is passed to the function by default?

Reply:
yep